programs/ipsec/ipsec.in

```diff
@@ -758,7 +758,14 @@ ipsec_import() {
 	    exit 1
 	fi
 
-	pk12util -i "${pkcs12bundle}" -d "${IPSEC_NSSDIR_SQL}"
+	# First try blanc password to avoid uselessly prompting interactively
+	pk12util -i "${pkcs12bundle}" -d "${IPSEC_NSSDIR_SQL}" -W '' 2>/dev/null
+	# check for SEC_ERROR_BAD_PASSWORD
+	if [ $? -eq 18 ]; then
```

Comment:
This condition doesn't look correct, as I see:
```console
$ pk12util -d sql:$PWD/../n -i tests/cert-tests/data/openssl.p12 -W ''
pk12util: PKCS12 decode not verified: SEC_ERROR_BAD_PASSWORD: The security password entered is incorrect.
$ echo $?
17
```
According to the pk12util.c, the exit code doesn't exactly correspond to SEC_ERROR_*, but we can expect 17 (= PK12UERR_DECODEVERIFY) when password is incorrect.